lib/tasks/test.rake

```diff
@@ -16,4 +16,36 @@ namespace :test do
     t.verbose = true
     t.warning = false
   end
+
+  namespace :syntax do
+    desc "Validate selected template snapshots against ksvalidator with EL7 profile"
+    task :kickstart_el7 do
+      Dir.glob('test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/*Kickstart*').each do |file|
+        puts "Validating #{file}"
+        `ksvalidator --version RHEL7 "#{file}"`
+        abort("invalid kickstart: #{file}") unless $?.success?
```

Comment:
Why did you choose backtits over `system` here, when you don't work with the output?
`system` would give you the result without touching global variables:

```suggestion
        result = system("ksvalidator --version RHEL7 '#{file}'")
        abort("invalid kickstart: #{file}") unless result
```

Reply:
Well `$?` is thread local, but indeed looks nicer.